### DISCUSSION TITLE
Is it possible to run multiple cores?

### DISCUSSION THREAD

**Discussion by minosss** [tauri-apps/tauri#4970] [General]
like the browser private window Independent of the default window. I want to login multiple users in my app

**Comment by FabianLars** [comment]
You can isolate windows by giving them unique user data dirs: https://docs.rs/tauri/latest/tauri/window/struct.WindowBuilder.html#method.data_directory - Doesn't have an effect on macOS though iirc.
Note that this, at least on windows, does increase memory usage because webview2 shares processes only if they have the same user data dir.

If that doesn't work for you, you'd need to spawn multiple instances of the app (you can't spawn mutliple apps/cores in the same main.rs file). Though again, this may not work on macos, because it has a built-in single instance mechanism and idk how to work around that for this usecase.
